.vscode/launch.json

```diff
@@ -4,14 +4,6 @@
   // environment or development setup.
   "version": "0.2.0",
   "configurations": [
-    {
-      "type": "rdbg",
-      "name": "Debug current file with rdbg",
-      "request": "launch",
-      "script": "${file}",
-      "args": [],
-      "askParameters": true
-    },
```

Comment:
Is removing this part of doing Square stuff? Or is it incidental?

Reply:
Oh not sure what happened here. Oops.